tools/reviewer/web/src/app/dashboard/diff/diff-discussion/diff-discussion.component.html

```diff
@@ -0,0 +1,53 @@
+<!-- Title -->
+<div class="cr-titlebar">
+  <div class="title">
+    <span class="bold">Discussion</span>
+  </div>
+  <div class="buttons" fxFlex>
+    <button
+      class="cr-light-button"
+      mat-raised-button
+      color="primary"
+    >Expand</button>
+  </div>
+</div>
+<div class="cr-header">
+  <mat-table [dataSource]="threads">
+      <ng-container matColumnDef="discussions">
+        <mat-header-cell *matHeaderCellDef>
+          General discussions
+          ({{getAmountOfComments()}} comments,
+          {{getAmountOfComments()}} unresolved)
```

Comment:
same function to get the comments and unresolved comments?

Reply:
Do we have a way to know an amount of unresolved *comments*?

Reply:
yeah I think there was. Thread has a property isDone. I think I discussed it with Ofer some time back. We should ask @oferb just to be sure.

Reply:
IsDone is a field of a thread, not comment.
https://github.com/google/startup-os/blob/master/tools/reviewer/service/code_review.proto#L105

Reply:
Not sure I follow, but indeed IsDone is a thread field, and so unresolved is a thread-related status. You could say the last comment in an unresolved thread is unresolved.